### ISSUE TITLE
Strange behaviour with direct route code completion

### ISSUE THREAD

**Issue by fharms** [camel-tooling/camel-idea-plugin#484]
I'm testing with IDEA 2019.1 and see this when I make code completion with the Camel DSL
```java
 @Override
    public void configure() {

        from("file:mailOutbox?bridgeErrorHandler=true")
                .routeId("test2")
                .process(exchange -> System.out.printf(""))
                .convertBodyTo(String.class)
                .
                .to("direct:hest");

        from("direct:hest")
                .to("direct:noop");
    }
```
![code-completion-direct-route](https://user-images.githubusercontent.com/1123285/56350650-8fc7c480-61cb-11e9-9830-c584e76ddea8.gif)

**Comment by fharms** [comment]
@rastislavpapp are you seeing this also ?

**Comment by rastislavpapp** [comment]
Yes, I've managed to reproduce this behavior. I'll fix it.

**Comment by fharms** [comment]
@rastislavpapp awesome. I have sent you an invitation so we are able to assign issues to you :-)

**Comment by rastislavpapp** [comment]
Solved by #488
